assignments/closure.js

```diff
@@ -1,21 +1,55 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function makeFunc() {
+  let name = 'May';  
+  let count = 0;
+  function displayName() {
+    count += 1;  //keeps track of times invoked
+    console.log(name+count);  //name and original count can be accessed because it is in a larger scope
+  }
+  return displayName; 
+}
+
+const myFunc = makeFunc();
+myFunc();
+myFunc();
+myFunc();
+
```

Comment:
this is definitely correct. you are using closures because you're returning a function that is making use of variables outside it's scope.  to make it more useful, I would pass in name into makefunc, and you can see closures in action:  

![image](https://user-images.githubusercontent.com/30554629/59484921-67f68500-8e39-11e9-9784-3c76b7306d84.png)

Reply:
![image](https://user-images.githubusercontent.com/30554629/59484953-865c8080-8e39-11e9-9233-cc0976302a94.png)